### PR TITLE
new phidget board

### PR DESCRIPTION
**Pull request by benmaidel**
- new serial for new phidget board + sensor naming for battery_light_monitor

### CODE REVIEW

**Comment by benmaidel** [comment]
fine to merge

**Comment by fmessmer** [comment]
as it is hardware specs...merging

File: cob_hardware_config/raw3-3/config/phidgets.yaml [hunk 1]
```diff
@@ -3,13 +3,13 @@ update_mode: polling                #switch between poll and event driven proces
 
 boards:
   ifk_general:
-    serial_num: 285651
+    serial_num: 272849
     ratiometric: true
     sensors:
-      bat1:
+      voltage:
```
Comment (destogl) [inline cob_hardware_config/raw3-3/config/phidgets.yaml:9]: Why not rename this for phidget boards on all robots?

Comment (fmessmer) [inline cob_hardware_config/raw3-3/config/phidgets.yaml:9]: well...before #400 was merged, raw3-3 was the only robot affected by this parameter name change...

@ipa-bnm can you please clarify whether the related PR (in cob_voltage_control? in cob_phidgets?) already has been merged?

I would kindly ask @destogl to send another PR for raw3-5 once @ipa-bnm confirmed that this name change has made it to ipa320/indigo_dev branch

**Comment by destogl** [comment]
@ipa-fxm will do!
